### ISSUE TITLE
When rescheduling a booking someone else made with me, let me override available times and book anytime I want

### ISSUE THREAD

**Issue by jdaviescoates** [calcom/cal.com#11842]
### Is your proposal related to a problem?

Sometimes I want to reschedule a booking to a time that technically isn't available in my calendar (because there is an unconfirmed placeholder event, or an event I'd like to attend but don't need to, or whatever) and I still want to be able to reschedule to that time.

### Describe the solution you'd like

I'd like the option/ ability to override availability times when reschedule a booking with me to be able to choose *any* time, not just the time other people booking with me can choose

**Comment by ciaranha** [comment]
We have some upcoming features including being able to access a troubleshooter from your public link and being able to edit availability on a calendar weekly view. This screenshot below doesn't have the exact solution to what you're asking for, but I think there's probably a nice way for us to allow you to book times on slots that are technically unavailable and to make it easy to see when would be good to do that.

![CleanShot 2023-11-13 at 10 33 20@2x](https://github.com/calcom/cal.com/assets/4536123/e2667242-0a76-4b2e-a388-7a6ca4ff3dad)

**Comment by jbritten** [comment]
+1 for this request. At present it's a headache to reschedule in Cal.com when a customer emails me 5 minutes before the start of a meeting that they want to push back an hour. In Cal.com, I just want to click reschedule and be able to pick any time without regard to the "availability" that Cal.com thinks I have.

**Comment by xcellentavi** [comment]
> +1 for this request. At present it's a headache to reschedule in Cal.com when a customer emails me 5 minutes before the start of a meeting that they want to push back an hour. In Cal.com, I just want to click reschedule and be able to pick any time without regard to the "availability" that Cal.com thinks I have.

@jbritten Agreed, on a Friday I get requests to move things around, I just want to be able to quickly customize those last minute changes and not have to temporarily revise availability and buffer limits in settings, do the booking, and then go back and revert them.  Looking forward to seeing an override option to alleviate this process.

**Comment by xcellentavi** [comment]
@PeerRich, I know this is a low-priority feature, but I thought I'd check to see if there is any update on the implementation.

Thank you!

**Comment by cubistico** [comment]
I definitely need this too! Should not be low priority, as I'm getting into this issue quite frequently.
